src/client/signupComponent.js

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var SignupComponent = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <header>
```

Comment:
Header is not part of the signup components Header is the Header component. The SignupComponent will be rendered into the same Component as the Header, but the Header is not the part of the SignupComponent. You dont need to create a header component it will be created in another story, and when it is done we can render it into the application